Redoublet-frontend/src/views/Multiplayer/MultiplayerTableJoinView.vue

```diff
@@ -186,7 +188,8 @@ th {
   background-color: white;
   border: 5px solid purple;
   padding: 20px;
-  z-index: 9999; /* Make the prompt appear on top of everything */
+  z-index: 9999;
+  /* Make the prompt appear on top of everything */
```

Comment:
Ff commentje terugzetten op juiste regel
```suggestion
  z-index: 9999; /* Make the prompt appear on top of everything */
```